platform-sdk/base-crypto/src/main/java/module-info.java

```diff
@@ -22,7 +22,7 @@
     requires transitive com.swirlds.logging;
     requires transitive org.hiero.base.concurrent;
     requires transitive org.hiero.base.utility;
-    requires transitive lazysodium.java;
+    requires transitive com.goterl.lazysodium;
```

Comment:
This is the official name now:  https://github.com/terl/lazysodium-java/pull/122